### DISCUSSION TITLE
Can be too tall if Pico board is socketed

### DISCUSSION THREAD

**Discussion by linker3000** [frntc/SIDKick-pico#19] [General]
I found that with a socket ftted to plug in the Pico board, the whole assembly was too tall for my C64C and made contact with the bottom of the keyboard. I couldn't close the C64C's top!

I fixed this by soldering the pico directly to the SIDKick board using header pins. 

Maybe using a very low profile socket for the Pico and short, round-pin headers would have been OK.

**Comment by frntc** [comment]
Hi, 

yes, depending on what you stack up, it might end up being too high :-) Inexpensive round low-profile headers can be found on LCSC and Aliexpress. But since the Pico (or a clone) does not cost more than the ICs on the other side, why not solder it directly?

**Comment by linker3000** [comment]
Thanks for the quick reply. 

Yes, soldering directly is an option. Originally, I wanted to have one fully socketed SIDKick carrier board so I could use it to program and test other Pico boards without having to make up whole units and inserting/removing them from the C64 DIP socket, thus stressing that socket too much.
